docs/design.md

```diff
@@ -260,14 +262,15 @@ runs encounters data that necessitate conflict resolution.
 
 When a transaction restarts, it changes its priority and/or moves its
 timestamp forward depending on data tied to the conflict, and the
-transaction begins anew, updating its intents. Since the set of keys
+transaction begins anew reusing the same tx id. Since the set of keys
 being written change between restarts, a set of keys written during
-prior attempts at the transaction is maintained by the client. As it
-restarts the transaction from the beginning, it removes keys from this
-set as it writes them again. The remaining keys--should the transaction
-run to completion--are crufty write intents which must be deleted
-*before* the transaction commit record’s status is set to COMMITTED.
-Many transactions will have no keys in this set.
+prior attempts at the transaction is maintained by the client as a set of
+dirty keys. As it replays the transaction from the beginning, it 
```

Comment:
It's just a way to describe this rather than calling it a set of keys. It's
worth calling it something.

On Fri, May 22, 2015 at 2:14 PM Tamir Duberstein <notifications@github.com>
wrote:

> In docs/design.md
> <https://github.com/cockroachdb/cockroach/pull/1113#discussion_r30918232>:
>
> >  being written change between restarts, a set of keys written during
> > -prior attempts at the transaction is maintained by the client. As it
> > -restarts the transaction from the beginning, it removes keys from this
> > -set as it writes them again. The remaining keys--should the transaction
> > -run to completion--are crufty write intents which must be deleted
> > -*before* the transaction commit record’s status is set to COMMITTED.
> > -Many transactions will have no keys in this set.
> > +prior attempts at the transaction is maintained by the client as a set of
> > +dirty keys. As it replays the transaction from the beginning, it
>
> what's a set of dirty keys? this implies some sort of type, but i don't
> think that's what's intended
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/1113/files#r30918232>.
>